code/modules/research/designs/equipment_designs.dm

```diff
@@ -280,3 +280,14 @@
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 5000, MAT_PLASMA = 2500, MAT_TITANIUM = 500, MAT_BLUESPACE = 500)
 	category = list("Equipment")
+
+/datum/design/jawsoflife_rescue
+	name = "Rescue Jaws of Life"
+	desc = "The Rescue variant of the Jaws of Life, designed to quickly open doors in emergencies. Bigger and less versatile than the original."
+	id = "jawsoflife"
```

Comment:
Shares an ID with the actual jaws, it’s probably be better to name this something like “rescuejaws” to avoid confusion

Reply:
then it can be used for data levels even if they already deconstructed jaws of life